js-extensions/spec/ResourceLoadTracker-spec.js

```diff
@@ -3,6 +3,8 @@ var expect = require('chai').expect;
 
 var ResourceLoadTracker = new (require('../dist/ResourceLoadTracker').default)();
 
+var noop = () => {};
```

Comment:
Why do you define that here and not on e.g. in  `ResourceLoadTracker.onMount('ep-1', noop);` where you then use a fallback when needed.

```
...
onMount(a, callback = () => { this._renderAllExtensions() }) {
   if(callback) {
    callback()
  }
    ...
}
```

Then you can simply do, `ResourceLoadTracker.onMount('ep-1', null);`

much cleaner and much more versatile

Reply:
no, i want to make sure it's passed in, it's needed

Reply:
Then you should test for that in `onMount` and throw an error if not passed.